targets/ipq806x

```diff
@@ -1,7 +1,29 @@
-ATH10K_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+# The QCA9980 is discontinued from Qualcomm. It didn't receive any firmware-update since >4 years.
+# Any kind of meshing was never implemented for this chip. It is (most likely) broken forever.
+# The QCA9984 on the other hand works fine for 11s meshes on both bands.
 
+QCA9980_PACKAGES='-kmod-ath10k kmod-ath10k-ct -ath10k-firmware-qca99x0 ath10k-firmware-qca99x0-ct'
+QCA9984_PACKAGES='kmod-ath10k -kmod-ath10k-ct ath10k-firmware-qca9984 -ath10k-firmware-qca9984-ct'
 
-# TP-Link
 
+#
+# QCA9980
+#
+
+# TP-Link
 device tp-link-archer-c2600 tplink_c2600
-packages $ATH10K_PACKAGES
+packages $QCA9980_PACKAGES
+
+
+#
+# QCA9984
+#
+
+# -- ToDo --
+# The BROKEN flag for QCA9984 can be removed when switching to an OpenWrt base newer than 18.06,
+# as in 18.06 the LEDs are not yet functional. This is already fixed in the current OpenWrt master.
```

Comment:
If the LEDs are currently broken for the radio add a BROKEN flag around QCA9984 and unflag the target as BROKEN in general, because the reason in `targets.mk` only reflects on QCA9980.

Reply:
As no device in this target is currently fully functional, i would leave the target as a whole broken. For easier unbreaking however, i would add a BROKEN clause explicitly for the QCA9980. Sounds good?

Reply:
But the broken remark for the target does not apply for QCA9984, it is misleading at best now.

https://github.com/freifunk-gluon/gluon/blob/master/targets/targets.mk#L25

Reply:
Ok, unbreaking the target seems like the wrong move here, sorry. But updating the remark seems to be in order.